docs/user-manual/en/client-reconnection-failover.md

```diff
@@ -1,10 +1,32 @@
-# Client Reconnection and Session Reattachment
+# Client Reconnection and Failover
```

Comment:
Can we simplify the doc here?


I still haven't figured out how you would configure the client to failover to another server... 

I thought about just having two topics would make it simpler.




- Failover to a backup Server


(don't worry too much about mentioning Apache ActiveMQ bblablablabla all the time)

just say, if you want to reconnect to a backup server, configure this, and that.





- Failover to non backup servers

Mention what you have to do if you want to failover to non backup servers...





There are so many options here that's getting confusing to understand.


that's my only concern here.

Reply:
@clebertsuconic thanks for your feedback, I'll rewrite the doc

Reply:
@clebertsuconic I partially rewrote the doc following your suggestions, could you take a look?